packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -98,21 +100,38 @@ const checkH264Support = (sdp) => {
  */
 const validateSdp = (sdp) => {
   const parsedSdp = sdpTransform.parse(sdp);
-  parsedSdp.media.forEach((mediaLine) => {
+  let validSdp = true;
+   parsedSdp.media.forEach((mediaLine) => {
+     let result = false;
     if (mediaLine.candidates && mediaLine.candidates.length === 0) {
       LoggerProxy.logger.error('PeerConnectionManager:index#validateSdp --> iceCandidate: Ice candadate never completed');
-      return false;
+      validSdp = validSdp && false;
+      return;
+    } else {
+      // Sometimes the candidates might be there but only IPV6 we need to makes sure we have IPV4
+      let hostCandidate = mediaLine.candidates.filter((candidate) =>{
+         return !!(candidate.type === HOST && candidate.ip.match(IPV4_REGEX))
+        } );
```

Comment:
eslint hasn't formatted these files correctly, and looking at the file as a whole show's that eslint has been disabled
Remove the `eslint-disable` at the top of the file and have eslint autofix the file